### ISSUE TITLE
[OTHER] denojs should be called deno

### ISSUE THREAD

**Issue by PantheraRed** [devicons/devicon#1546]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### What would you like to share?

Originally, **Deno** does not have a suffix "js". But in devicons it is named `denojs`. In my opinion, the suffix "js" should dropped with respect to its original name `deno`.

### Additional information

# Sources
https://deno.land/
https://github.com/denoland/deno

**Comment by Snailedlt** [comment]
Agreed, like I mentioned in your PR #1526 this will be a breaking change, so I added the major label to it

**Comment by Panquesito7** [comment]
Would implementing some sort of backward-compatibility icon names work in this case?

**Comment by Snailedlt** [comment]
@Panquesito7 Yes, but it would increase the complexity of devicons without much gain. I suggest we wait until the next major release to fix this, since it's not a crucial thing
